syntax = "proto3";

package Ydb.BSConfig.V1;

option java_package = "com.yandex.ydb.bsconfig.v1";
option java_outer_classname = "BSConfigGrpc";
option java_multiple_files = true;

import "ydb/public/api/protos/ydb_bsconfig.proto";

service BSConfigService {

    // Initialize Blobstorage host configs and box
    rpc ReplaceStorageConfig(BSConfig.ReplaceStorageConfigRequest) returns (BSConfig.ReplaceStorageConfigResponse);

    // Fetch Blobstorage host configs and box
    rpc FetchStorageConfig(BSConfig.FetchStorageConfigRequest) returns (BSConfig.FetchStorageConfigResponse);

}